datarobot_provider/example_dags/datarobot_datetime_partitioning_pipeline_dag.py

```diff
@@ -0,0 +1,69 @@
+# Copyright 2023 DataRobot, Inc. and its affiliates.
+#
+# All rights reserved.
+#
+# This is proprietary source code of DataRobot, Inc. and its affiliates.
+#
+# Released under the terms of DataRobot Tool and Utility Agreement.
+from datetime import datetime
+
+from airflow.decorators import dag
+from datarobot import AUTOPILOT_MODE
+from datarobot.enums import DATETIME_AUTOPILOT_DATA_SELECTION_METHOD
+
+from datarobot_provider.operators.ai_catalog import UploadDatasetOperator
+from datarobot_provider.operators.autopilot import StartAutopilotOperator
+from datarobot_provider.operators.datarobot import CreateProjectOperator
+from datarobot_provider.sensors.datarobot import AutopilotCompleteSensor
+
+
+@dag(
+    schedule=None,
+    start_date=datetime(2022, 1, 1),
+    tags=['example', 'datetime_partitioning'],
+    params={
+        "dataset_file_path": "/train_datetime.csv",
+        "project_name": "test airflow datetime-partitioning project",
+        "autopilot_settings": {
+            "target": "y",
+            "mode": AUTOPILOT_MODE.QUICK,
+        },
+        "datetime_partitioning_settings": {
+            "use_time_series": False,
+            "datetime_partition_column": 'datetime',
+            "number_of_backtests": 1,
+            "autopilot_data_selection_method": DATETIME_AUTOPILOT_DATA_SELECTION_METHOD.DURATION,
+            "validation_duration": "P1Y0M0D",
+            "gap_duration": "P0Y0M0D",
```

Comment:
I'm not familiar with these `P0Y0M0D` values. Probably people new to the DR API would be unfamiliar too... maybe adding a link to our public docs describing these values?

Reply:
Added methods from datarobot.helpers.partitioning_methods to construct duration string like: construct_duration_string(years=1),